src/renderer/components/notification-comp.tsx

```diff
@@ -160,6 +160,9 @@ export default class NotificationComp extends React.Component<{}, IState> {
         const { color, flash } = data;
         data.color = (color && !color.startsWith('#')) ? '#' + color : color;
         this.setState(data as IState);
+        if (this.flashTimer) {
```

Comment:
@KiranNiranjan This is just an improvement. If there is a `this.flashTimer` do not run this part [line](https://github.com/symphonyoss/SymphonyElectron/pull/752/files#diff-5e55b4298403f384ef25fde3a18882caR166)
So this will prevent from flashing again if we push a new notification.

Reply:
@keerthi16 That's correct but what if there is the same tag with a different color? So, it will always be better to reset.